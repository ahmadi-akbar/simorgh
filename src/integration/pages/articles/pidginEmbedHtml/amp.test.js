/**
 * @service pidgin
 * @pathname /pidgin/articles/c39rjygpmv1o
 */

// not sure this will work
import { runAmpEmbedHtmlTests } from '../../../common';

describe('AMP', () => {
  describe(pageType, () => {
    runAmpEmbedHtmlTests();
  });
});
