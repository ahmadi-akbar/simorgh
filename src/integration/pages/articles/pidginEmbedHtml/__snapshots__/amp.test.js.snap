// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AMP Articles Embed HTML should match snapshot 1`] = `
<div
  class="bbc-koled1-embedDiv"
  data-e2e="embed-error"
>
  <strong
    class="bbc-pgllns-Text"
  >
    Sorry, we canâ€™t display this part of the story on this lightweight mobile page.
  </strong>
  <div
    class="bbc-l03aby-errorLinkWrapper"
  >
    <a
      class="bbc-1gbmkx-self-inlineLink"
      href="http://localhost:7080/pidgin/articles/c39rjygpmv1o"
    >
      View the full version of the page to see all the content.
    </a>
  </div>
</div>
`;
